modules/ml/src/main/java/org/apache/ignite/ml/composition/boosting/GDBTrainer.java

```diff
@@ -165,7 +165,7 @@ public GDBTrainer(double gradStepSize, Integer cntOfIterations,
 
         try (Dataset<EmptyContext, DecisionTreeData> dataset = builder.build(
             new EmptyContextBuilder<>(),
-            new DecisionTreeDataBuilder<>(featureExtractor, lbExtractor)
+            new DecisionTreeDataBuilder<>(featureExtractor, lbExtractor, false)
```

Comment:
How do you decide when to use `false` and when `true` in these example? Is it a random?

Reply:
Yes, in random manner)